packages/owl-bot/src/configs-store.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {OwlBotLock, OwlBotYaml} from './config-files';
+
+export interface Configs {
+  // The body of .Owlbot.lock.yaml.
+  lock: OwlBotLock | undefined;
```

Comment:
It's a little more conventional to list these as `lock?: OwlBotLock`

Reply:
Done.